tests/Custom/DesiredCapabilitiesTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
+
+use Behat\Mink\Driver\Selenium2Driver;
+use Behat\Mink\Tests\Driver\TestCase;
+
+class desiredCapabilitieTest extends TestCase
+{
+
+    public function testGetDesiredCapabilties()
+    {
+        $driver = $this->getSession()->getDriver();
+        $this->assertNotEmpty($driver->getDesiredCapabilities(), 'desiredCapabilities empty');
```

Comment:
Where would I instantiate the driver?  It appears that it's already instantiated once phpunit hits the DesiredCapabilitiesTest class, and that's done from \Behat\Mink\Driver\TestCase.